tcms/testplans/tests/test_new_plan.py

```diff
@@ -82,10 +113,10 @@ def _test_plan_create_new(self, is_active):
         self.assertEqual(response.status_code, HTTPStatus.FOUND)
 
         plan = TestPlan.objects.get(
-            name=self.plan_name,
+            name=self.request["name"],
```

Comment:
Either use `self.plan_name` or hard-code the value in `self.request`. No need to use both at the same time b/c `plan_name` isn't used anywhere else as-is.